src/redux/actions/samples/createSampleFile.js

```diff
@@ -23,17 +23,25 @@ const createSampleFile = (
     const url = `/v2/experiments/${experimentId}/samples/${sampleId}/sampleFiles/${type}`;
     const body = {
       sampleFileId,
-      size: fileForApiV1.size,
+      size: file.size,
     };
 
+    // Leaving out path, errors
+    // They are used during the upload process, not redux
+    // TODO we should check if they can be separated somehow between
+    // The ones that are relevant for the api vs
+    // the ones that are only necessary for retry (fileObject, compressed)
+    // Perhaps into an uploadRetryParams object or something
```

Comment:
This is something I kind of wanted to do on this PR too, but perhaps it would be good to get this group of main changes in first.